src/provider/ehr_launch.jl

```diff
@@ -84,17 +84,25 @@ function provider_ehr_launch(config::ProviderEHRLaunchConfig;
         redirect = false,
     )
     headers = Dict(authorize_response.headers)
+
     let 
-        auth_error_msg = string(
+        error_msg = string(
             "Something when wrong when authenticating to the EHR. ",
             "One possible explanation is that your `redirect_uri` value does not exactly ",
             "match any of the `redirect_uri` values that are on file with the EHR.",
         )
-        haskey(headers, "Location") || throw(ErrorException(auth_error_msg))
+        haskey(headers, "Location") || throw(ErrorException(error_msg))
     end
     location = headers["Location"]
     location_uri = URIs.URI(location)
     location_queryparams = URIs.queryparams(location_uri)
+
+    let
+        error_msg = "Encountered an error while trying to authenticate to the EHR.",
```

Comment:
@DilumAluthge There is a trailing comma here so `error_msg` ends up as a tuple. This results in the cryptic error message
```julia
┌ Error: Error: invalid_grant
└ @ SMARTAppLaunch ~/.julia/packages/SMARTAppLaunch/JoeYY/src/provider/ehr_launch.jl:102
┌ Error: Error: Invalid launch token
└ @ SMARTAppLaunch ~/.julia/packages/SMARTAppLaunch/JoeYY/src/provider/ehr_launch.jl:103
ERROR: MethodError: no method matching convert(::Type{AbstractString}, ::Tuple{String, Nothing})
```

Reply:
https://github.com/JuliaHealth/SMARTAppLaunch.jl/pull/28